scalastyle-config.xml

```diff
@@ -111,7 +111,7 @@
    <parameter name="maxMethods"><![CDATA[30]]></parameter>
   </parameters>
  </check>
- <check level="warning" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="true"></check>
- <check level="warning" class="org.scalastyle.file.NewLineAtEofChecker" enabled="true"></check>
- <check level="warning" class="org.scalastyle.file.NoNewLineAtEofChecker" enabled="false"></check>
+ <check level="warning" class="org.scalastyle.scalariform.PublicMethodsHaveTypeChecker" enabled="false"/>
```

Comment:
This will explicitly detect the override keyword. So for instance:

```
  def receive = {...}
```

will get a warning. While

```
override def receive = {...}
```

comes out clean. Is there a way to make it more intelligent to also mean implementing a field from a trait?